CefGlue.Common.Shared/Serialization/CefValueSerialization.cs

```diff
@@ -2,34 +2,37 @@
 using System.Collections.Generic;
 using System.Dynamic;
 using System.Text.Json;
+using System.Text.Json.Serialization;
 
 namespace Xilium.CefGlue.Common.Shared.Serialization
 {
     internal static class CefValueSerialization
     {
-        private const int SerializerMaxDepth = byte.MaxValue;
-        
+        private const int SerializerMaxDepth = int.MaxValue;
+        private const int DeserializerMaxDepth = byte.MaxValue;
```

Comment:
theres no connection with byte... so lets use a const
```suggestion
        private const int DeserializerMaxDepth = 255;
```